src/database/models/reviews.js

```diff
@@ -35,6 +35,10 @@ module.exports = (sequelize, DataTypes) => {
 				type: DataTypes.STRING,
 				primaryKey: true,
 			},
+			notes: {
```

Comment:
@adithyadinesh0412 check the character limit of note. If its more than 256 make it text

Reply:
its set to 200. @priyanka-TL